com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/NetworkProfilerOverride.cs

```diff
@@ -0,0 +1,249 @@
+using System;
+using System.Collections.Generic;
+using MLAPI.Profiling;
+using Unity.Profiling;
+using UnityEditor;
+using UnityEngine;
+
+namespace ProfilerModuleOverride
+{
+    [InitializeOnLoad]
+    public class NetworkProfilerOverride
+    {
+#if UNITY_2020_2_OR_NEWER
+        const string RPCModuleName = "MLAPI RPCs";
+        const string OperationModuleName = "MLAPI Operations";
+        const string MessageModuleName = "MLAPI Messages";
+
+        [System.Serializable]
+        public class MLAPIProfilerCounter
+        {
+            public string m_Name;
+            public string m_Category;
+        }
+
+        [System.Serializable]
+        public class MLAPIProfilerModuleData
+        {
+            public MLAPIProfilerModuleData()
+            {
+                m_ChartCounters = new List<MLAPIProfilerCounter>();
+                m_DetailCounters = new List<MLAPIProfilerCounter>();
+            }
+
+            public List<MLAPIProfilerCounter> m_ChartCounters;
+            public List<MLAPIProfilerCounter> m_DetailCounters;
+            public string m_Name;
+        }
+
+        [System.Serializable]
+        public class MLAPIModules
+        {
+            public List<MLAPIProfilerModuleData> m_Modules;
+        }
+
+        private static List<MLAPIProfilerCounter> CreateRPCCounters()
+        {
+            var list = new List<MLAPIProfilerCounter>();
+            list.Add(new MLAPIProfilerCounter() { m_Name = ProfilerConstants.NumberOfRPCsSent.ToString(), m_Category = ProfilerCategory.Network.Name });
```

Comment:
nit, this might be more readable if you add a constructor to MLAPIProfilerCounter

Reply:
Changed this into a lambda being called.